ppc64le_build.bash

```diff
@@ -1,22 +1,50 @@
 #!/bin/bash
 
 echo "#####################"
-echo "IT IS RECOMMENDED TO RUN THIS BUILD SCRIPT ON UBUNTU BIONIC!"
+echo "IT IS RECOMMENDED TO RUN THIS BUILD SCRIPT ON UBUNTU XENIAL OR FEDORA 30!"
 echo "#####################"
 echo "If any error occurs, please refer to https://wiki.raptorcs.com/wiki/Porting/Chromium for missing dependencies or others."
 echo "#####################"
 
 set -eux
 
-#apt-get update
-#apt-get install -y build-essential clang git vim cmake python libcups2-dev pkg-config libnss3-dev libssl-dev libglib2.0-dev libgnome-keyring-dev libpango1.0-dev libdbus-1-dev libatk1.0-dev libatk-bridge2.0-dev libgtk-3-dev libkrb5-dev libpulse-dev libxss-dev re2c subversion curl libasound2-dev libpci-dev mesa-common-dev gperf bison uuid-dev clang-format libatspi2.0-dev libnotify-dev libgconf2-dev libcap-dev libxtst-dev libxss1 python-dbusmock openjdk-8-jre ninja-build clang-format wget
+. /etc/os-release
+OS=$NAME
+OSVER=$VERSION_ID
+
+if [[ $OS == "Ubuntu" ]] && [[ $OSVER == "16.04" ]]
```

Comment:
While it's good you choose to add support for Ubuntu Xenial, can't we keep Ubuntu Bionic even if it couldnt build Chromium lately?

Reply:
Thanks for the review @leo-lb! The reason why I didn't add bionic was that it had build issues and I couldn't really get it to build/test. But as you said, I'll just let bionic build too to enter there.

Reply:
"enter here" you mean in the Xenial code path? Dependencies may be different. - Just add a condition that retains the old code that installed dependencies, is there any problem with that?

Reply:
OK, it seems that I had misinterpreted your comment. I'll add separate condition for bionic. Also, the issue is not with the dependencies, I've used mostly the same dependencies for xenial (except for python3.7 and ninja). @NishikantThorat had tried the bionic build and had faced issues in the build, so we didn't really pursue it further.

Reply:
I was able to build on bionic too.